lib/game.rb

```diff
@@ -38,10 +46,26 @@ def switch_players
   def ask_move(player)
     output("#{player.name}'s move", true)
     output('Enter your next move (1-9): ')
-    move = input
+    move = input.to_i
+    until valid_move?(move)
+      output('Invalid move. Try again: ')
+      move = input.to_i
+    end
+    move
   end
 
-  def won?; end
+  def valid_move?(move)
```

Comment:
This method too is better associated with the Player class rather than the game class